src/gap-skipper.js

```diff
@@ -0,0 +1,88 @@
+import Ranges from './ranges';
+
+let adaptiveSeeking;
+let gapSkipper;
+let seekingObject;
+
+gapSkipper = function(options) {
+  this.ready(() => {
+    seekingObject = { player: this,
```

Comment:
I wouldn't store a reference to player or tech here I would just use them directly.